src/internal/proc.js

```diff
@@ -151,6 +151,19 @@ function createTaskIterator({ context, fn, args }) {
       )
 }
 
+function getLocation(instrumented) {
+  return (
+    // TODO move symbol creation to redux-saga side
+    instrumented[Symbol.for('babel-plugin-transform-redux-saga-source')] || instrumented.__source || null
```

Comment:
NOTE: we can add symbol no to extend saga effect descriptor. the most simple first step is add global symbol

Reply:
well, it could also always be importable from the `redux-saga`, so we dont have to rely on globals

Reply:
agree, I've even checked how [lodash-webpack-plugin](how https://github.com/lodash/lodash-webpack-plugin/blob/master/src/index.js#L48-L86) does it. but haven't tried to apply that technique.

Reply:
Hm, I think it would really be a matter of inserting an import for appropriate export from redux-saga package. It seems that lodash is doing some more magic there in the quoted code - for some other reasons.